src/services/federated_account.ts

```
@@ -1,9 +1,10 @@
 import { FederatedProvider, User } from "@prisma/client";
 import axios from "axios";
 import { EnvConfig } from "utils/env";
-import { BadRequestError } from "@utils/error";
-import console from "console";
+import { BadRequestError, InternalServerError, KnownErrorCode } from "@utils/error";
 import { ExtendedPrismaClient } from "@utils/prisma";
+import { hash } from "crypto";
```

Biome (1.9.4)

[error] A Node.js builtin module should be imported with the node: protocol.

Using the node: protocol is more explicit and signals that the imported module belongs to Node.js.
Unsafe fix: Add the node: protocol.


(lint/style/useNodejsImportProtocol)